frontend/setupChelonia.js

```diff
@@ -55,16 +55,20 @@ const setupChelonia = async (): Promise<*> => {
     const identityContractID = await sbp('gi.db/settings/load', SETTING_CURRENT_USER)
     if (!identityContractID) return
     await sbp('chelonia/reset', cheloniaState)
+    // If there's an active session, we need to start capture now
+    await sbp('swLogs/startCapture', identityContractID)
```

Comment:
Can we double-check here that everything related to this function call is correct?

I.e. this comment is a bit strange, setting up an either/or statement, with both branches saying the same thing:

```js
// This function is tasked with most common tasks related to setting up Chelonia
// for Group Income. If Chelonia is running in a service worker, the service
// worker should call this function. On the other hand, if Chelonia is running
// in the browsing context, the service worker is the one that should call this
// function.
const setupChelonia = async (): Promise<*> => {
```

And the other place this `'swLogs/startCapture'` is called is conditionally in `identity.js`. Do we need a condition here too?

```js
     // If running in a SW, start log capture here
      if (typeof WorkerGlobalScope === 'function') {
        await sbp('swLogs/startCapture', identityContractID)
      }
```

Reply:
> setting up an either/or statement, with both branches saying the same thing:

Right, the comment should say 'the browsing context is the one that should call this' instead of 'the service worker is the one that should call this'. I.e., it could be called by either the SW or the browser (although the latter case hasn't been tested).

`swLogs/startCapture'`: Maybe you're right this should be a conditional call like it is in `identity.js`.